test/orbits/keplerian.jl

```diff
@@ -0,0 +1,467 @@
+            "m": m,
+        }
+    """
+    small_star = py"small_star"(orbit.period, orbit.t0, orbit.aR_star, orbit.incl, orbit.ecc, orbit.omega)
+
+    # Compare
+    t = small_star["t"]
+    r_batman = small_star["r_batman"]
+    m = small_star["m"]
+    r = compute_r(orbit, t)
+    @test sum(m) > 0
+    @test allclose(r_batman[m], r[m], atol=2e-5)
+end
+
+@testset "KeplerianOrbit: impact" begin
+    # Sample model from `Transits.jl`
+    orbit = KeplerianOrbit(
+        R_star = 0.189,
+        M_star = 0.151,
+        P = 0.4626413,
+        t0 = 0.2,
+        b = 0.5,
+        ecc = 0.8,
+        omega = 0.1,
+    )
+
+    pos = relative_position.(orbit, orbit.t0)
+    @test hypot(pos[1], pos[2]) ≈ orbit.b
+end
+
+@testset "KeplerianOrbit: flip" begin
+    orbit = KeplerianOrbit(
+        M_star = 1.3,
+        R_star = 1.1,
+        t0 = 0.5,
+        period = 100.0,
+        ecc = 0.3,
+        incl = 0.25*π,
+        omega = 0.5,
+        Omega = 1.0,
+        M_planet = 0.1,
+    )
+
+    orbit_flipped = flip(orbit, 0.7)
+
+    t = range(0, 100; length=1_000)
+
+    u_star = as_matrix(_star_position.(orbit, orbit.R_star, t))
+    u_planet_flipped = as_matrix(_planet_position.(orbit_flipped, orbit.R_star, t))
+    for i in 1:3
+        @test allclose(u_star[:, i], u_planet_flipped[:, i], atol=1e-5)
+    end
+
+    u_planet = as_matrix(_planet_position.(orbit, orbit.R_star, t))
+    u_star_flipped = as_matrix(_star_position.(orbit_flipped, orbit.R_star, t))
+    for i in 1:3
+        @test allclose(u_planet[:, i], u_star_flipped[:, i], atol=1e-5)
+    end
+end
+
+@testset "KeplerianOrbit: flip circular" begin
+    t = range(0, 100; length=1_000)
+
+    orbit = KeplerianOrbit(
+        M_star = 1.3,
+        M_planet = 0.1,
+        R_star = 1.0,
+        P = 100.0,
+        t0 = 0.5,
+        incl = 45.0,
+        ecc = 0.0,
+        omega = 0.5,
+        Omega = 1.0
+    )
+    orbit_flipped = flip(orbit, 0.7)
+
+    u_star = as_matrix(_star_position.(orbit, orbit.R_star, t))
+    u_planet_flipped = as_matrix(_planet_position.(orbit_flipped, orbit.R_star, t))
+    for i in 1:3
+        @test allclose(u_star[:, i], u_planet_flipped[:, i], atol=1e-5)
+    end
+
+    u_planet = as_matrix(_planet_position.(orbit, orbit.R_star, t))
+    u_star_flipped = as_matrix(_star_position.(orbit_flipped, orbit.R_star, t))
+    for i in 1:3
+        @test allclose(u_planet[:, i], u_star_flipped[:, i], atol=1e-5)
+    end
+end
+
+@testset "KeplerianOrbit: compute_aor" begin
+    duration = 0.12
+    period = 10.1235
+    b = 0.34
+    r = 0.06
+    R_star = 0.7
+    aor = compute_aor(duration, period, b, r=r)
+
+    for orbit in [
+        KeplerianOrbit(
+            period=period, t0=0.0, b=b, a=R_star * aor, R_star=R_star
+        ),
+        KeplerianOrbit(
+            period = period,
+            t0 = 0.0,
+            b = b,
+            duration = duration,
+            R_star = R_star,
+            r = r,
+        ),
+        ]
```

Comment:
```suggestion
            KeplerianOrbit(
                period=period, t0=0.0, b=b, a=R_star * aor, R_star=R_star
            ),
            KeplerianOrbit(
                period = period,
                t0 = 0.0,
                b = b,
                duration = duration,
                R_star = R_star,
                r = r,
            ),
        ]
```

Reply:
hmm, this actually gives me 1 tab = 8 spaces. Was this intended?